test/reducers/measures_test.js

```diff
@@ -0,0 +1,23 @@
+import { expect } from '../test_helper';
+
+import { flattenCategories, flattenMeasures } from '../../src/reducers/measures';
+
+describe('Measure reducers', () => {
+  it('should flatten categories', () => {
```

Comment:
I doubt this is the case, but this PR modifies the package.json. Maybe try an npm install?